dev/publish_release.sh

```diff
@@ -67,6 +70,34 @@ function curl_error() {
   fi
 }
 
+
+#
+# Publishing Apache Zeppelin artifact to Apache snapshot repository.
+#
+function publish_snapshot_to_maven() {
+  cd "${WORKING_DIR}/zeppelin"
+  echo "Deploying Apache Zeppelin $RELEASE_VERSION version to snapshot repository."
+
+  if [[ ! $RELEASE_VERSION == *"SNAPSHOT"* ]]; then
+    echo "ERROR: Snapshots must have a version containing 'SNAPSHOT'"
+    echo "ERROR: You gave version '$RELEASE_VERSION'"
+    exit 1
+  fi
+
+  mvn versions:set -DnewVersion=$RELEASE_VERSION
+  tmp_settings="tmp-settings.xml"
+  echo "<settings><servers><server>" > $tmp_settings
+  echo "<id>apache.snapshots.https</id><username>$ASF_USERID</username>" >> $tmp_settings
+  echo "<password>$ASF_PASSWORD</password>" >> $tmp_settings
+  echo "</server></servers></settings>" >> $tmp_settings
+
+  mvn --settings $tmp_settings -DskipTests $PUBLISH_PROFILES -Drat.skip=true deploy
```

Comment:
this seems slightly different from publish_to_maven() below - should it `-Pbeam`?  `-Dmaven.repo.local`?